### PR TITLE
Fixes #11763 - Add inherited parameters to Host view api

### PR DESCRIPTION
**Pull request by shlomizadok**

### CODE REVIEW

File: app/models/host/managed.rb [hunk 1]
```diff
@@ -462,6 +462,16 @@ def host_params
     @cached_host_params = hp
   end
 
+  def host_inherited_params_objects
```
Comment (domcleal) [inline app/models/host/managed.rb:467]: Why is this duplicating the code above?

Comment (shlomizadok) [inline app/models/host/managed.rb:467]: Code above returns hash of the inherited params
This method returns the objects.
I we don't care about the objects, I can use the hashes from the code above.

Comment (domcleal) [inline app/models/host/managed.rb:467]: Sure, but they're implementing exactly the same logic - that should be avoided and combined.

Comment (shlomizadok) [inline app/models/host/managed.rb:467]: refactored

**Comment by dLobatog** [comment]
Merged as 0a8128420a64f236244e3fa8a18e0a923857f66d, thanks @shlomizadok!

**Comment by dLobatog** [comment]
I've opened [11858](http://projects.theforeman.org/issues/11858) to keep on improving this method, there's a bit of useless duplication there still. Thanks again.

**Pull request closed without merging**